api/resource/resource.proto

```diff
@@ -33,6 +33,7 @@ message Metadata {
   google.protobuf.Timestamp created = 8;
   google.protobuf.Timestamp updated = 9;
   repeated string finalizers = 6;
+  map<string, string> labels = 10;
```

Comment:
Huh? Should't it be in labels PR?

Reply:
Talos has its own gRPC API for COSI for historical reasons. I'm not really introducing labels into Talos here, but this change is required to make the code compile (as Metadata proto structure should implement COSI interface).

Reply:
so this change is from the recently added label selectors in COSI, right?

Reply:
yep, as COSI got bumped, it has now labels in COSI gRPC API (vs. Talos gRPC resources API), so it requires Talos Metadata proto to have these labels